nca/NetworkConfig/PeerContainer.py

```diff
@@ -37,6 +37,14 @@ def __eq__(self, other):
                 and self.services == other.services
         return False
 
+    def is_comparable_with_other_container(self, other):
+        """
+        Two peer containers are considered comparable if having same set of peers, regardless their namespaces
+         and services
+         :param PeerContainer other: the peer container to compare current with
+        """
+        return isinstance(other, PeerContainer) and self.peer_set == other.peer_set
+
```

Comment:
I think we should consider the peer containers as comparable only if both peer_set and services are identical (thus it is comparable regardless of namespaces only).

For example, if an Ingress resource uses some service name, but the service is different between the two configs (captures different pods for example), then this common Ingress policy has different connectivity results on each config, and this cannot fall into the category of "have the same network topology and the same set of policies"